pkgs/development/python-modules/bluepy-devices/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, bluepy
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "bluepy-devices";
```

Comment:
The pname can be bluepy_devices but the name in all packages cannot.

Reply:
Thanks, changed to `bluepy_devices`.